test/recipes/30-test_evp_data/evppkey_ffdhe.txt

```diff
@@ -105,6 +105,8 @@ CEKAlg=id-aes128-wrap
 Ctrl = dh_pad:1
 SharedSecret=89A249DF4EE9033B89C2B4E52072A736D94F51143A1ED5C8F1E91FCBEBE09654
 
+# FIPS 3.0.0 has a bug in the default padding #17859
```

Comment:
I think the fix involves not being able to set the padding at all i.e. This test tries to set the padding to 0 via a ctrl that it ignores.

Reply:
Updated the comment.